mingus/core/scales.py

```diff
@@ -612,3 +1150,130 @@ def ascending(self):
+
+    def __init__(self, note, octaves=1):
+        """Create the SUHMM lydian scale starting on the chosen note."""
+        super(SuperUltraHyperMegaMetaLydian, self).__init__(note, octaves)
+        self.name = '{0} SUHMM lydian'.format(self.tonic)
+
+    def ascending(self):
+        tonic = self.tonic
+        notes = []
+        for i in range(self.octaves*2):
+            notes += [reduce_accidentals(note) for note in list(Lydian(tonic).ascending()[:4])] # 4 creates overlap on the fifth
+            tonic = intervals.perfect_fifth(tonic)
+        
+        if self.octaves == 1:
+            return notes
+        else:
+            return notes[:1-self.octaves]
+        
+    def descending(self):
+        tonic = self.tonic
+        notes = []
+        for i in range(self.octaves*3):
+            notes += [reduce_accidentals(note) for note in list(Lydian(tonic).descending()[:3])] # 3 = 7 - 4
+            tonic = intervals.perfect_fifth(tonic)
+        
+        if self.octaves == 1:
+            return notes[:-1]
+        else:
+            return notes[:-1-self.octaves]  
+        
+    def generate(self, i):
+        print('Please use ascending() and descending() methods instead of generate() for SUHMM scales')
+        return False
+
+class SuperUltraHyperMegaMetaMixolydian(_Scale):
+
+    """The super-ultra-hyper-mega-meta mixolydian scale.
+    
+    This is the sound of continually darkening around the cycle of fifths (getting 'more mixolydian'), 
+    in both ascending and descending forms.
+
+    Example:
+    >>> print(SuperUltraHyperMegaMetaMixolydian('C'))
+    Ascending:  C D E, F G A, Bb C D, Eb F G, ... 
+    Descending: C Bb A G, F Eb D C, Bb Ab G F, Eb Db C Bb, ...
+    """
+
+    type = 'other'
+
+    def __init__(self, note, octaves=1):
+        """Create the SUHMM mixolydian scale starting on the chosen note."""
+        super(SuperUltraHyperMegaMetaMixolydian, self).__init__(note, octaves)
+        self.name = '{0} SUHMM mixolydian'.format(self.tonic)
+
+    def ascending(self):
+        tonic = self.tonic
+        notes = []
+        for i in range(self.octaves*3):
+            notes += [reduce_accidentals(note) for note in list(Mixolydian(tonic).ascending()[:3])]
+            tonic = intervals.perfect_fourth(tonic)
+        
+        if self.octaves == 1:
+            return notes[:-1]
+        else:
+            return notes[:-self.octaves]
+        
+    def descending(self):
+        tonic = self.tonic
+        notes = []
+        for i in range(self.octaves*2):
+            notes += [reduce_accidentals(note) for note in list(Mixolydian(tonic).descending()[:4])]
+            tonic = intervals.perfect_fourth(tonic)
+        
+        if self.octaves == 1:
+            return notes
+        else:
+            return notes[:-self.octaves]  
+        
+    def generate(self, i):
+        print('Please use ascending() and descending() methods instead of generate() for SUHMM scales')
+        return False
```

Comment:
It seems that returning `False` would totally break the expectations of the caller. I think that you should either raise an exception or accept to run the default `generate` normally. (In the latter case you could issue a warning, using `warnings.warn()`)